test/unit/api/api.test.js

```diff
@@ -168,10 +169,7 @@ describe('API', () => {
     beforeEach(() => {
       nock('https://api.github.com').post('/graphql').reply(200, successfulAuthResponseWrite);
 
-      models = td.replace('../../../lib/models', {
-        Subscription: td.object(['getAllForInstallation', 'findOrStartSync']),
-        Installation: td.object(['getForHost']),
-      });
+      models = td.replace('../../../lib/models');
```

Comment:
Because `test/setup/app.js` calls `td.replace` the dictionary passed here never worked, but that's okay because `app.js` implemented all but the `getForHost` which we were using `jest` to override anyways. Fixed this up to avoid tripping someone else up next time

Reply:
Our mock-heavy tests make me a little nervous about the feedback our tests give for this sort of change. But I imagine our manual testing would show errors pretty quickly if we botched something. We don't have many models.

Reply:
So, I was step debugging to figure this out. The `Subscription/Installation` part of this call is completely ignored by testdouble. The original testdouble call within the setup folder is the one that happens first, and is the only one that ever occurs.